users.html

```diff
@@ -245,6 +254,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Chongqing Institute of Meteorological Sciences
   L.marker([29.555700242810225, 106.53819106830466], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Chongqing Institute of Meteorological Sciences</strong><br>Chongqing, China<ul><li><a href='https://www.cma.gov.cn/' target='_new'>Website</a></li></ul>");
 
+  // Chongqing University
+  L.marker([29.56488930697949, 106.46817209791666], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Chongqing University</strong><br>Chongqing, China<ul><li><a href='http://www.cqu.edu.cn/' target='_new'>Website</a></li></ul>");
```

Comment:
Page is in Chinese but there may not be an English option.  It is what it is.

Reply:
Now fixed in https://github.com/geoschem/geoschem.github.io/pull/44/commits/6c1cebc8fd9ab1a5bc5d41edae948c86303de382